pkgs/tools/X11/xosview2/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, xorg }:
+
+stdenv.mkDerivation rec {
+    name = "xosview2-${version}";
+    version = "2.2.2";
+
+    src = fetchurl {
+        url = "https://sourceforge.net/projects/xosview/files/${name}.tar.gz/download";
+        sha256 = "3502e119a5305ff2396f559340132910807351c7d4e375f13b5c338404990406";
+        name = "${name}.tar.gz";
+    };
+
+    doCheck = false;
+
+    buildInputs = [ xorg.libX11 ];
+
+    meta = {
+        description = "";
+        homepage = "http://xosview.sourceforge.net/index.html";
+        license = stdenv.lib.licenses.gpl1;
+        maintainers = [ stdenv.lib.maintainers.seanz ];
+        platforms = stdenv.lib.platforms.all;
+    };
+}
```

Comment:
Use 2 space indentation for the file to be consistent with other nix expressions in nixpkgs.